specreduce/wavelength_calibration.py

```diff
@@ -191,6 +191,10 @@ def model(self, new_model):
         self._clear_cache()
         self._model = new_model
 
+    @property
+    def fit_residuals(self):
```

Comment:
probably should have a docstring to explain what this is and that its only available after calling `apply_to_spectrum`.

Reply:
done

Reply:
oh actually, it would be accessible after calling either .wcs or .apply_to_spectrum. the fit is done in .wcs.